fastestimator/pipeline.py

```diff
@@ -179,7 +179,7 @@ def benchmark(self, mode: str = "train", epoch: int = 1, num_steps: int = 1000,
             if self.batch_size:
                 log_interval = log_interval * self.batch_size
 
-            print("\nBreakdown of time taken by Pipeline Operations:")
+            print("\nBreakdown of time taken by Pipeline Operations: Epoch: {}, Mode: {}".format(epoch, mode))
```

Comment:
Pipeline Operations (train epoch 7):

rather than

Pipeline Operations: Epoch 7: Mode: train

Reply:
Okay. I will update that.